doc/source/data/doc_code/pytorch_quick_start.py

```diff
@@ -0,0 +1,39 @@
+# flake8: noqa
+# isort: skip_file
+# fmt: off
+
+# __pt_quickstart_load_start__
+import ray
+import numpy as np
+
+
+dataset = ray.data.from_numpy(np.ones((1, 100)))
+# __pt_quickstart_load_end__
+
+
+# __pt_quickstart_model_start__
+class TorchPredictor:
+    import torch
+    import torch.nn as nn
```

Comment:
```suggestion
```

Reply:
Moved this line to above the class header because otherwise, the example doesn't work.